engine/language-server/src/main/scala/org/enso/languageserver/text/CollaborativeBuffer.scala

```diff
@@ -307,7 +307,7 @@ class CollaborativeBuffer(
         Some(CapabilityRegistration(CanEdit(bufferPath)))
       else
         None
-    sender ! OpenFileResponse(Right(OpenFileResult(buffer, writeCapability)))
+    sender() ! OpenFileResponse(Right(OpenFileResult(buffer, writeCapability)))
```

Comment:
Why? Has something changed in scalac?

Reply:
Yep! A method that has a param list must now be called with a param list.

Reply:
Thanks, I hate it